lib/mail_room/arbitration.rb

```diff
@@ -0,0 +1,14 @@
+module MailRoom
+  module Arbitration
+    def self.[](name)
```

Comment:
Can this be made a `module_function`? http://apidock.com/ruby/Module/module_function Not a huge deal, I've just been trying to be better about it.